paper/abstract.md

```diff
@@ -0,0 +1,20 @@
+
+
+As higher education decision makers increasingly turn to statistical
+and machine learning algorithms to predict student success, they often
+do so without a clear understanding of how these algorithms work or
+the tradeoffs made when basing decisions upon their predictions. Of
+particular concern are hidden biases inherent in an uncritical use of
+these algorithms that negatively affect already marginalized student
```

Comment:
I might leave out the already in front of marginalized

Reply:
I'm good with that.

Reply:
Removed in updated version